rust/build_error.rs

```diff
@@ -13,7 +13,7 @@
 //! functions could still be called in the runtime).
 
 #![no_std]
-#![feature(const_panic, core_panic)]
+#![feature(core_panic)]
```

Comment:
rust-lang/rust#88954 is in Rust 1.57 so `core_panic` isn't needed anymore. The panic below can be written as `panic!("{}", msg);`

Reply:
Yeah, @bjorn3 also mentioned it in the issue. Let's do also the `vec!` disabling here then.